examples/sample-app/README.md

```diff
@@ -349,9 +349,18 @@ the ip address shown below with the correct one for your environment.
             # Optional: pre-pull the router image.  This will be pulled automatically when the pod is created but will
             # take some time.  Your pod will stay in Pending state while the pull is completed
             $ docker pull openshift/origin-haproxy-router
+            
+            # Create a service account that the router will use.  This service account must have access to use a
+            # security context constraint that allows host ports
+            $ echo '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"router"}}' | oc create -f -
+            
+            # Edit your security context constraint to add the new service account in the users section
+            # in the form of system:serviceaccount:<namespace>:<name>.  In the above example the full
+            # name would be system:serviceaccount:default:router if you are creating the router in the default namespace.
+            # oc edit scc <name>
 
```

Comment:
The sample app /could/ use the privileged SCC however, for the most part,  we've been leaving these types of things intentionally ambiguous in docs to prevent copy/paste issues that leave people with an unsecure cluster.  That's why I just put <name>.